shop/src/main/java/ro/msg/learning/shop/controller/ProductServiceController.java

```diff
@@ -0,0 +1,56 @@
+package ro.msg.learning.shop.controller;
+
+import ro.msg.learning.shop.dto.ProductDTO;
+import ro.msg.learning.shop.mapper.ProductMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.model.Product;
+import ro.msg.learning.shop.service.ProductService;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+public class ProductServiceController {
+    @Autowired
+    ProductService productService;
+    @Autowired
+    ProductMapper productMapper;
+
+    @GetMapping("/products")
```

Comment:
You can annotate your controller with the **@RequestMapping** annotation and give it your API path value, which is '/products'.  In this manner, you would not need to repeat yourself for each request method.